### ISSUE TITLE
Cleanup Archived Projects

### ISSUE THREAD

**Issue by hbetts** [conventional-changelog/conventional-changelog#276]
@stevemao would it be possible for me to get the proper permissions to close out all opened issues and pull requests for archived projects within the `conventional-changelog-archived-repos` org?

Looks like it's still possible for people to push commits to a PR associated with an archived project, which leads to notification e-mails (As an example, I received a notification e-mail for conventional-changelog-archived-repos/conventional-changelog-angular/pull/17)

**Comment by stevemao** [comment]
I already gave you permissions. I can't close that PR myself since it's archived. The best way to do is unsubscribe it.

**Comment by stevemao** [comment]
Seems like we have to unarchive it to close an issue. Even for admins...

**Comment by hbetts** [comment]
Thank you @stevemao. I went through, un-archived the repositories, closed issues and pull requests (with a note that they have been merged into the mono-repository), and then re-archived the repositories.

There should no longer be any open issues or pull requests against archived repositories.

Hopefully closing those issues and pull requests will encourage those developers to re-open them on this mono-repository, where we can accept them, and publish them for consumption by the community.
